src/Main.php

```diff
@@ -97,6 +108,15 @@ public function declare_wc_hpos_compatibility() {
 		}
 	}
 
+	/**
+	 * Declare Product Editor compatibility.
+	 */
+	public function declare_product_editor_compatibility() {
+		if ( class_exists( '\Automattic\WooCommerce\Utilities\FeaturesUtil' ) ) {
+			\Automattic\WooCommerce\Utilities\FeaturesUtil::declare_compatibility( 'product_block_editor', 'postnl-for-woocommerce/postnl-for-woocommerce.php', true );
```

Comment:
```suggestion
			\Automattic\WooCommerce\Utilities\FeaturesUtil::declare_compatibility( 'product_block_editor', POSTNL_WC_PLUGIN_BASENAME, true );
```

Can we use this instead, since its a published in WP.org with different folder name

Reply:
@Abdalsalaam, no problem! Thank you for the suggestion!